src/controllers/v1.controller.ts

```diff
@@ -284,6 +292,42 @@ export class V1Controller {
     return new HttpErrors.InternalServerError('Application not found')
   }
 
+  async checkClientStickiness(
+    rawData: object
+  ): Promise<{ preferredApplicationID: string; preferredNodeAddress: string; rpcID: number }> {
+    // Parse the raw data to determine the lowest RPC ID in the call
+    const parsedRawData = Object.keys(rawData).length > 0 ? JSON.parse(rawData.toString()) : JSON.stringify(rawData)
+    const rpcID = parseRPCID(parsedRawData)
+
+    if (rpcID > 0) {
```

Comment:
Bit of a nitpick, but I feel this would be a bit easier to read if we invert this if—If it's 0 or below, just return the default response in line `328`

Reply:
Disagree -- I also need the failure case at the bottom in case there is an issue inside of the if {}. For instance if the redis key is not found, it needs a failure path.